extension/encoding/awslogsencodingextension/internal/unmarshaler/elb-access-log/elb.go

```diff
@@ -299,6 +300,10 @@ func convertTextToALBAccessLogRecord(fields []string) (ALBAccessLogRecord, error
 		TargetStatusCodeList:   fields[26],
 		Classification:         fields[27],
 		ClassificationReason:   fields[28],
+		ConnectionTraceID:      fields[29],
+		TransformedHost:        fields[30],
+		TransformedURI:         fields[31],
+		RequestTransformStatus: fields[32],
```

Comment:
Not sure why these were missed before. @piffall  do you think these are new fields ?

Anyway, I think it might be better to perform an array index check before assigning these new extended fields. And this is to avoid any downstream additions from failing. For example, unit tests & integration tests that lack these fields.

Reply:
https://docs.aws.amazon.com/elasticloadbalancing/latest/application/doc-history.html mentions:

> This release adds support to transform host headers and URLs for incoming requests before the load balancer routes the traffic to a target.

That was was added on October 15. So I'm guessing they're new fields.

We _could_ change the `if fieldsCount < 29 {` error message above, but then the extension won't be able to parse older access logs.

I would be inclined  to leave that as-is, and do the index check as @Kavindu-Dodan suggests. If the number of fields is too few, we could treat that the same as the field being "unknownField".